src/schema/imported/manifest.json

```diff
@@ -780,6 +780,16 @@
           "default": "*",
           "description": "Maximum version of Gecko to support. Defaults to '*'. (Requires Gecko 45)",
           "pattern": "^[0-9]{1,3}(\\.[a-z0-9*]+)+$"
+        },
+        "install_origins": {
```

Comment:
@diox given that it looks that we now got to an agreement about the pending issue related to this property [^1], and that I'm already reviewing the patches on the Firefox side, I though to run the schema imports for you and push them into my fork (in a [branch named `pre-review/install-origins`](https://github.com/mozilla/addons-linter/compare/master...rpl:pre-review/install-origins)) to make it easier for you to cherry pick what is needed directly from there, the one that may be more directly useful for your work in this pull request are:
- https://github.com/mozilla/addons-linter/commit/b94bba48e4559fdba9deb26293eb359da3d8d859 (this one is just the schema import of the "install_origins" definition from [D129105](https://phabricator.services.mozilla.com/D129105), other schema unrelated to `install_origins' is in separate patches in that same fork, hopefully you will only need this one to try our your changes)
- https://github.com/mozilla/addons-linter/commit/720066a79a03bdb64d13b1274cc5084d27bca927 (this one contains the small tweak that we will need to add to `src/schema/updates/manifest.json`, plus the changes to `src/schema/updates/manifest.json` got by re-importing the schema again on top of the current version of this patch, which means that it does remove this definition and instead adds "maxItems" to the one added by the schema data import from the other patch + a small tweak to the test case to confirm that everything else works as expected)

Let me know how it goes when you will give it a look to cherry picking what's needed from that branch (especially if there is something that isn't really clear or if it isn't working for you).

I'm going to mark the other comment (https://github.com/mozilla/addons-linter/pull/4018/files/0d3278bc1b2fbaad5431df465d19930830757ac3#r745572036) as resolved.

[^1]: in particular: where the new property should be placed (top level manifest property in the `ManifestBase` schema), its manifest property name (`install_origins`) and what should be the format expected to be enforced (`origin` as defined by the specs), I did

Reply:
@diox here is a patch for the strategy that we have been discussing over Matrix today (I did rerun the import from the current beta on top of the change to the updates/manifest.json file, and tweaked the test case from your PR to confirm that they all pass when the install_origins is moved to the top level):
https://github.com/mozilla/addons-linter/commit/3df53d6fc6802aae252b57266620d9f448119127